lib/db.js

```diff
@@ -721,6 +766,16 @@ module.exports = function (
     return this.pool.post('/account/' + account.uid.toString('hex') + '/verifyEmail')
   }
 
+  DB.prototype.verifyTokens = function (tokenVerificationId, tokenData) {
+    log.trace({ op: 'DB.verifyToken', tokenVerificationId: tokenVerificationId })
+    return this.pool.post('/tokens/' + tokenVerificationId.toString('hex') + '/verify',
+        unbuffer(
+          tokenData,
+          'inplace'
+        )
+      )
+  }
```

Comment:
I recommend adding a test for it explicitly, that's a lot of layers in between the effect we want, and the code we expect to trigger it.